perun/templates/scripts/traces_table_js.html.jinja2

```diff
@@ -131,7 +153,7 @@
                 paging: false,
                 info: false,
                 searching: false,
-                order: [[3, "desc"]]
+                order: [[5, "desc"]]
```

Comment:
Not sure how constant-friendly is the JS here, but maybe extracting these columns to variables would be safer.